scripts/CompareFitMinimizers/results_output.py

```diff
@@ -269,31 +269,31 @@ def display_name_for_minimizers(names):
     return display_names
 
 
-def calc_cell_len_rst_table(columns_txt, items_link):
+def calc_cell_len_rst_table(columns_txt, items_link, cells, color_scale=None):
     """
-    Calculate what width in ascii characters we need for an RST table.
+    Calculate ascii character width needed for an RST table, using the length of the longest table cell.
 
     @param columns_txt :: list of the contents of the column headers
+    @param items_link :: the test names used for summary tables
```

Comment:
Do you mean "@param items_link :: the test names used for an table"? I.e. that items_link contains the names for just one table not all the fit benchmarking tables.